common/logisticspipes/gui/guidebook/book/MenuItem.java

```diff
@@ -0,0 +1,127 @@
+package logisticspipes.gui.guidebook.book;
+
+import java.util.ArrayList;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.audio.PositionedSoundRecord;
+import net.minecraft.client.audio.SoundHandler;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+
+import lombok.Getter;
+
+import logisticspipes.LPConstants;
+import logisticspipes.gui.guidebook.GuiGuideBook;
+import logisticspipes.utils.GuideBookContents;
+import logisticspipes.utils.string.StringUtils;
+
+public class MenuItem {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	// Getting constants
+	private final int z$titleButtons = 15; // Title and Buttons Z
+	private final int z$frame = 10; // Frame Z
+	private final int z$text = 5;// Text/Information Z
+	private final int z$background = 0;  // Background Z
+
+	//// Information storage
+	@Getter
+	private GuideBookContents.Chapter chapter;
+
+	//// Drawing variables
+	public boolean visiible, hovering, enabled;
+	private int btn$bgX0, btn$bgY0, btn$bgX1, btn$bgY1;
+	private int btn$x0, btn$y0, btn$x1, btn$y1, btn$x2, btn$y2, btn$x3, btn$y3;
+
+	//// Atlas
+	// Btn
+	private final int atlas$btn$bgU0 = 64, atlas$btn$bgV0 = 32, atlas$btn$bgU1 = 96, atlas$btn$bgV1 = 64;
+	private final int atlas$btn$u0 = 0, atlas$btn$v0 = 64, atlas$btn$u1 = 2, atlas$btn$v1 = 66, atlas$btn$u2 = 14, atlas$btn$v2 = 78, atlas$btn$u3 = 16, atlas$btn$v3 = 80;
```

Comment:
Don't use `$` in variable names in most cases, it's ugly and mostly used only by generated stuff

Reply:
Sure